.github/workflows/review.yaml

```diff
@@ -0,0 +1,105 @@
+#
+# Check for the Squid Project Merge Process criteria
+# published at https://wiki.squid-cache.org/MergeProcedure#votes
+#
+name: PR Approval
+
+on:
+  schedule:
+    - cron: "10 1 * * *" # once a day
```

Comment:
Why not more often? Say every 4 hours or so?

Reply:
The criteria calculations have a 1-day resolution. So it is not worth doing more than once per day after the initial check.